### PR TITLE
Some 0.7.0 changes

### PR DESCRIPTION
**Pull request by asgerius**

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Removed `log.tqdm`
 - Added binning functions to `pelutils.ds.plot`
+- Added `pelutils.ds.distributions` with `scipy` distributions that use same notation as Jim Pitman's "Probability"
```
Comment (sorenmulli) [inline HISTORY.md:7]: Please shorten commit message. Eventually by writing Pitman instead of Jim Pitman

File: examples/ds/plot.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+""" Plotting examples using pelutils.ds.plot """
+import matplotlib.pyplot as plt
+import numpy as np
+
+from pelutils.ds.plot import linear_binning, log_binning, normal_binning, get_bins, figsize_wide, rc_params_small, update_rc_params
+from pelutils.ds.distributions import norm, lognorm
+
+# Update params to make them fit nicely with the used figure size
+update_rc_params(rc_params_small)
```
Comment (sorenmulli) [inline examples/ds/plot.py]: Please put all example code in ifmain-block

**Review by sorenmulli — APPROVED** [review]
lgtm

File: HISTORY.md (hunk 1 above)
Comment (asgerius) [reply HISTORY.md:7]: What should I do in the meantime?

**Pull request merged**